docs/src/pages/[platform]/studio/create-form.page.tsx

```diff
@@ -0,0 +1,22 @@
+import { PathMeta } from '@/data/meta';
+import { Button } from '@aws-amplify/ui-react';
+import { useNavigationContext } from 'src/hooks/useNavigationContext';
+
+export const pageMeta: PathMeta = {
+  frontmatter: {
+    title: 'Create a form!',
+    description: `We're creating a form!!`,
+  },
+};
```

Comment:
Nice!

Reply:
I actually hit a wall on this so the export does nothing right now (had to add an entry to `meta.ts`)

Reply:
oh not nice!